Server/mods/deathmatch/logic/CStaticFunctionDefinitions.h

```diff
@@ -466,6 +466,7 @@ class CStaticFunctionDefinitions
     static bool AddColPolygonPoint(CColPolygon* pColPolygon, const CVector2D& vecPoint);
     static bool AddColPolygonPoint(CColPolygon* pColPolygon, uint uiPointIndex, const CVector2D& vecPoint);
     static bool RemoveColPolygonPoint(CColPolygon* pColPolygon, uint uiPointIndex);
+    static bool SetColPolygonHeight(CColPolygon* pPolygon, float fFloor, float fCeil);
```

Comment:
Nit: naming consistency
```suggestion
    static bool SetColPolygonHeight(CColPolygon* pColPolygon, float fFloor, float fCeil);
```